Reqnroll/Infrastructure/DefaultDependencyProvider.cs

```diff
@@ -117,7 +117,7 @@ public virtual void RegisterGlobalContainerDefaults(ObjectContainer container)
             container.RegisterTypeAs<FileBasedConfigurationResolver, IFormattersConfigurationResolver>("fileBasedResolver");
             container.RegisterTypeAs<EnvironmentConfigurationResolver, IFormattersEnvironmentOverrideConfigurationResolver>();
             container.RegisterTypeAs<FormattersConfigurationProvider, IFormattersConfigurationProvider>();
-            container.RegisterTypeAs<MessagesFormatter, ICucumberMessageFormatter>("messages");
+            container.RegisterTypeAs<MessageFormatter, ICucumberMessageFormatter>("messages");
```

Comment:
I think we also need to fix this at the registration and for the `Name` property as well.

Reply:
Done.